README.md

```diff
@@ -35,7 +35,15 @@ flutter pub get
 
 https://flutter.dev/docs/get-started/editor
 
-5. Run the App
+5. Generate Transient Files
+
+Files used for JSON decoding are generated using [`build_runner`](https://dart.dev/tools/build_runner).
+
+```
+flutter pub run build_runner build
```

Comment:
Maybe add a note for the watch option and how to delete conflicting files?

- https://pub.dev/packages/build_runner#built-in-commands
- https://pub.dev/packages/build_runner#command-line-options